src/gen_xml.ml

```diff
@@ -62,6 +62,11 @@ let start () = ref [], ref []
 let generate_code (x,_) index stmt =
   let name = choose_name stmt.props stmt.kind index in
   let input = Node ("in",[],params_to_values @@ params_only stmt.vars) in
+  let input_sets =
+    match inparams_only stmt.vars with
+    | [] -> None
+    | _ -> Some (Node ("in-set",[],params_to_values @@ inparams_only stmt.vars))
```

Comment:
I think I like `<in value= type= set=true>` more (where `set` is optional when false)